sw/host/opentitanlib/src/transport/ultradebug/spi.rs

```diff
@@ -42,9 +46,34 @@ impl UltradebugSpi {
             device: mpsse,
             inner: RefCell::new(Inner {
                 mode: TransferMode::Mode0,
+                cs_asserted_count: 0,
             }),
         })
     }
+
+    fn do_assert_cs(&self) -> Result<()> {
+        let device = self.device.borrow();
+        // Assert CS# (drive low).
+        device
+            .execute(&mut [mpsse::Command::SetLowGpio(
+                device.gpio_direction,
+                device.gpio_value & !Self::MASK_CHIP_SELECT,
+            )])
+            .context("FTDI error")?;
+        Ok(())
+    }
```

Comment:
Would this function be better as:
```
fn assert_cs(&mut self, assert: bool) -> Result<()>
```

Where the function does handles the reference counting and command construction based on the value of `assert`?

Reply:
I suppose that I could save some lines by consolidating the two internal methods into one.  I will do so, as I implement this API on the other transport backends.